website/server/controllers/api-v3/user.js

```diff
@@ -161,6 +176,10 @@ api.updateUser = {
       }
 
       if (acceptablePUTPaths[key]) {
+        if (key === 'flags.newStuff') {
+          val = _setNewStuffFlag(val, res);
+          key = 'flags.newStuffSeenDate';
```

Comment:
Shouldn't we also be setting newStuff to false. I'm not sure where we would use newStuffSeendDate if we just toggle newStuff. 

Also, maybe we should add analytics here to tracking them looking?